src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -119,7 +119,11 @@ const ActiveState = ({
     getUserProfileUrl,
 }: Props): React.Node => {
     const activeEntry = items.find(({ id, type }) => id === activeFeedEntryId && type === activeFeedEntryType);
+    const [selectedThreadParentId, setSelectedThreadParentId] = React.useState(null);
```

Comment:
Should this state tie into the "selected activity item" logic that the router currently handles? If not, we won't be able to support deep linking for replies and we'll likely wind up with multiple potentially unrelated activity cards in deep linking scenarios.

Reply:
It definitively should! I've missed it. I will double check if that's want we need and align on one solution!

Reply:
We decided not to touch the logic responsible for deep linking but update the visuals to match the selected state that Chuck (designer) prepared and aligned with the preview designers.

Reply:
I'm not sure I understand. The concern I expressed isn't a design issue, it's a data consistency issue in that this change will introduce a second source of truth for which item is "active". This seems to be related to @kkuliczkowski-box's recent [PR](https://github.com/box/box-ui-elements/pull/3134), as well.